README.rst

```diff
@@ -180,6 +182,8 @@ Let us know if you're writing a new one!
 .. _django-oscar-adyen: https://github.com/oscaro/django-oscar-adyen
 .. _django-oscar-przelewy24: https://github.com/kisiel/django-oscar-przelewy24
 .. _oscar-sagepay: https://github.com/udox/oscar-sagepay
+.. _django-oscar-payu: https://github.com/SalahAdDin/django-oscar-payu
```

Comment:
Is it fork of the https://bitbucket.org/spliceglobal/django-oscar-payu?

Reply:
Have i to put the original repository instead?